docs/www/data-transform.md

```diff
@@ -0,0 +1,393 @@
+
+To deploy the sample transform, run:
+
+`rpk wasm deploy uppercase/dist/main.js --name uppercase --description "Converts uppercase text to lowercase"`
+
+## Verify that the transform works
+
+After the transform is deployed, Redpanda processes every event produced to the source topic
+and runs the logic that is defined by the transform.
+
+To see our sample transform in action:
+
+1. Run: `rpk topic consume test_topic._result_`
+2. Produce events to the source topic:
+    a. In a second terminal run: `rpk topic produce test-topic`
+    b. Enter text and press CTRL+D to send the event to the source topic.
+3. In the terminal that shows the consumed events you’ll see the text that you produced but it is now formatted with uppercase characters.
+
+## Clean up the transform
+
+To stop a transform, you have to remove the transform from the cluster brokers.
+
+For our sample transform, run: `rpk wasm remove uppercase`
+
+The transform stops processing events and is removed.
+
+## Source code
+
+You can check what are all the possible export values in our [GitHub Page](https://github.com/vectorizedio/redpanda/blob/dev/src/js/modules/public/Coprocessor.ts).
+
+The (API package)[https://www.npmjs.com/package/@vectorizedio/wasm-api] is published under npm.
+
+## Final considerations
+
+Currently we only support Node.js but in the future, we're planning to add support for a complete WASM engine so that it can compile and run any WebAssembly binary on the fly. 
+
+### Running untrusted code
+
+We do not recommend you to run untrusted code. The reason for this is that the service will be running with the same privileges as Redpanda and with permission to read your data. 
+
+The golden rule here is to only run code where you thrust the authors or that you wrote yourself.
+
+### Under the hood
+
+We create a global control topic for each Redpanda cluster with the name `coproceessor_internal_topic`. This is done to support any Kafka client so you don´t *need* to use `rpk`. If anyone publishes to this topic, it will invoke a deployer implicitly. You just need to publish with the same headers and the correct format that we use. 
+
+### Scaling
+
+If you execute different types of transformations in different topics it's more advantageous to have multiple deployments. Remember that every time a transformation happens, you're subscribing to a topic and producing to another. Even though this process is really fast because it happens inside Redpanda, there's still a processing overhead. 
+
+### Tech Preview Caveats
+
+We don't use [Raft](https://raft.github.io/) to publish and replicate the data. So it's possible to have inconsistencies in some cases. For example, if you have a topic with a replication factor > 1 and you have stateful coprocessors that aren't idempotent producing different data onto these replication topics. 
```

Comment:
```suggestion
We don't use [Raft](https://raft.github.io/) to publish and replicate the data. The reason is because the same transform is applied locally on all nodes in y So it's possible to have inconsistencies in some cases. For example, if you have a topic with a replication factor > 1 and you have stateful or idempotent coprocessors producing different data onto these replication topics. 
```

Reply:
@graphcareful can you check again if this suggestion is correct? 
`The reason is because the same transform is applied locally on all nodes in y`  
looks like it's missing something

Reply:
sorry! `all nodes in the closer`